rpcd/playbooks/roles/rpc_support/defaults/main.yml

```diff
@@ -39,6 +39,10 @@ support_neutron_debug_packages:
   - arping
   - ethtool
 
+support_utility_pip_packages:
+  - os-client-config
```

Comment:
os-client-config can be removed here. It's already installed as part of shade's dependencies.